app/src/main/java/org/mozilla/tv/firefox/ScreenControllerStateMachine.kt

```diff
@@ -31,6 +32,7 @@ object ScreenControllerStateMachine {
                 Transition.ADD_OVERLAY
             }
             ActiveScreen.SETTINGS -> Transition.REMOVE_SETTINGS
+            ActiveScreen.FXA_PROFILE -> Transition.REMOVE_FXA_PROFILE
```

Comment:
Did Bram say we want to be able to close the profile with the menu button? I'm concerned we may later add functionality to this screen that requires the menu button and it won't be available (then again, with Android TV support potential, maybe it's not a problem).

Reply:
It's how the other settings screens worked, so I kept it in.  🤷‍♂ 

@brampitoyo do you think a menu button click from the FxA profile screen should return to the overlay, or would you like it to be a no-op in case we want to use it later?